back-end/api/quickstart/helper_test.py

```diff
@@ -60,4 +60,27 @@ def get_test_post_fields(i=1, author="testauthorid"):
     'pageSize': 20,
     'commentLink': 'link to comments',
     'comments': '{ "text": "nice test" }'
+  }
+
+# Private field used to test conditional
+# Note how `isPrivateToFriends` is False
+def get_test_public_post_fields(i=1, author="testauthorid"):
+  return {
+    '_id': i,
+    'title': f'testpost{i}',
+    'description': f'i am test post {i}',
+    'source': f'source post id{i}',
+    'origin': f'origin post id{i}',
+    'visibility': 'Public',
+    'unlisted': True,
+    'isPrivateToFriends': False,
+    'author': author,
+    'contentType': 'text/plain',
+    'content': 'Hello, I am a test post',
+    'categories': '["Testing"]',
+    'published': '03:25:53.827044',
+    'count': 5,
+    'pageSize': 20,
+    'commentLink': 'link to comments',
+    'comments': '{ "text": "nice test" }'
```

Comment:
Should use the get_test_post_fields method but just send in a visibility. I can add this in a future PR.